velox/functions/sparksql/Hash.cpp

```diff
@@ -347,9 +364,40 @@ std::shared_ptr<exec::VectorFunction> makeHash(
   return kHashFunction;
 }
 
-std::vector<std::shared_ptr<exec::FunctionSignature>> xxhash64Signatures() {
+std::shared_ptr<exec::VectorFunction> makeHashWithSeed(
+    const std::string& name,
+    const std::vector<exec::VectorFunctionArg>& inputArgs,
+    const core::QueryConfig& /*config*/) {
+  const auto& constantSeed = inputArgs[0].constantValue;
+  if (!constantSeed || constantSeed->isNullAt(0)) {
+    VELOX_USER_FAIL("{} requires a constant seed argument.", name);
```

Comment:
Would you clarify that seed cannot be null?

`VELOX_USER_FAIL("{} requires a constant non-null seed argument.", name);`